.github/workflows/iam-drift-detection.yml

```diff
@@ -0,0 +1,69 @@
+# Workflow for IAM drift detection
+# Approach:
+#   Run on schedule or manual
+#   Execute drift script
+#   If drift, create issue
+
+name: 'IAM drift detection'
+on:
+  workflow_call:
+    inputs:
+      organization-id:
+        required: true
+        type: string
```

Comment:
Commenting here, but please quote all of your yaml.

Reply:
Done.